TestApi/TestApi/Controllers/UtilitiesController.cs

```diff
@@ -60,7 +60,7 @@ public async Task<IActionResult> DeleteTestDataByPartialCaseTextAsync(DeleteTest
                 foreach (var hearingId in hearingIds)
                 {
                     await _videoApiClient.DeleteAudioApplicationAsync(hearingId);
-                    _logger.LogInformation($"Successfully deleted audio application with hearing id {hearingId}");
+                    _logger.LogInformation("Successfully deleted audio application with hearing id {hearingId}", hearingId);
                 }
             }
             catch (VideoApiException e)
```

Comment:
Expected to see some kind of logging, even it's being handled. It just fills in the gaps when picturing a scenario

Reply:
Added!